scripts/word-to-dtbook/src/main/resources/xml/TOC.xsl

```diff
@@ -0,0 +1,212 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="2.0"
+                xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                xmlns:w="http://schemas.openxmlformats.org/wordprocessingml/2006/main"
+                xmlns:pic="http://schemas.openxmlformats.org/drawingml/2006/picture"
+                xmlns:wp="http://schemas.openxmlformats.org/drawingml/2006/wordprocessingDrawing"
+                xmlns:dcterms="http://purl.org/dc/terms/"
+                xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                xmlns:cp="http://schemas.openxmlformats.org/package/2006/metadata/core-properties"
+                xmlns:dc="http://purl.org/dc/elements/1.1/"
+                xmlns:a="http://schemas.openxmlformats.org/drawingml/2006/main"
+                xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships"
+                xmlns:v="urn:schemas-microsoft-com:vml"
+                xmlns:m="http://schemas.openxmlformats.org/officeDocument/2006/math"
+                xmlns:dcmitype="http://purl.org/dc/dcmitype/"
+                xmlns:o="urn:schemas-microsoft-com:office:office"
+                xmlns:d="org.daisy.pipeline.word_to_dtbook.impl.DaisyClass"
+                xmlns="http://www.daisy.org/z3986/2005/dtbook/"
+                exclude-result-prefixes="w pic wp dcterms xsi cp dc a r v dcmitype d xsl m o xs">
+  <!--Parameter citation-->
+  <xsl:param name="Cite_style" select="d:Citation($myObj)"/>
+
+  <!--template for frontmatter elements-->
+  <xsl:template name="TableOfContents">
+    <xsl:param name="pagination" as="xs:string"/>
+    <xsl:message terminate="no">progress:Handling table of content</xsl:message>
```

Comment:
These "progress" messages are a problem because they end up in the job log. Converting a small file results in a lot of noise, e.g.

```
[INFO]     progress:Parsing document languages
[INFO]     progress:Document languages 42
[INFO]     progress:Checking for page styles
[INFO]     progress:Starting conversion of 26 elements
[INFO]     progress:Building the frontmatter
[INFO]     progress:Building the bodymatter
[INFO]     progress:Converting element w:tbl - 1 / 26
[INFO]     progress:Converting element w:p - 2 / 26
[INFO]     progress:Converting element w:p - 3 / 26
[INFO]     progress:Converting element w:tbl - 4 / 26
[INFO]     progress:Converting element w:p - 5 / 26
[INFO]     progress:Converting element w:p - 6 / 26
[INFO]     progress:Converting element w:p - 7 / 26
[INFO]     progress:Converting element w:p - 8 / 26
[INFO]     progress:Converting element w:tbl - 9 / 26
[INFO]     progress:Converting element w:p - 10 / 26
[INFO]     progress:Converting element w:p - 11 / 26
[INFO]     progress:Converting element w:p - 12 / 26
[INFO]     progress:Converting element w:p - 13 / 26
[INFO]     progress:Converting element w:p - 14 / 26
[INFO]     progress:Converting element w:p - 15 / 26
[INFO]     progress:Converting element w:p - 16 / 26
[INFO]     progress:Converting element w:p - 17 / 26
[INFO]     progress:Converting element w:p - 18 / 26
[INFO]     progress:Converting element w:p - 19 / 26
[INFO]     progress:Converting element w:p - 20 / 26
[INFO]     progress:Converting element w:p - 21 / 26
[INFO]     progress:Converting element w:p - 22 / 26
[INFO]     progress:Converting element w:p - 23 / 26
[INFO]     progress:Converting element w:p - 24 / 26
[INFO]     progress:Converting element w:p - 25 / 26
[INFO]     progress:Converting element w:sectPr - 26 / 26
```

Is there something we can do about this? Are the messages needed for the Word plugin? Could we use a different mechanism? Note that we have the [`pf:progress`](https://github.com/daisy/pipeline-modules/blob/8402acd2153f1f9aab6339a5f1526b05a7bb0a1d/common/common-utils/src/main/resources/xml/xslt/library.xsl#L121-L136) XPath function. Is that of interest?

Reply:
@Nico There is not much time left. Shall we comment out the messages for now?

Reply:
Sorry I am assigned to another project this week.

Yes you can comment them out if needed for now :
I'll keep them on the SaveAsDAISY module for progression report for the time being.

Reply:
OK

Reply:
Done